Makefile

```diff
@@ -18,20 +18,16 @@ cluster-down: delete-config delete-certs
 deploy-certs:
 	@echo 'Generating certs and deploying into active cluster...'
 	hack/webhook-create-signed-cert.sh --service trace-context-injector-webhook-svc --secret trace-context-injector-webhook-certs --namespace default
-	cat deploy/mutatingwebhook.yaml | hack/webhook-patch-ca-bundle.sh > deploy/mutatingwebhook-ca-bundle.yaml
+	cat deploy/base/mutatingwebhook.yaml | hack/webhook-patch-ca-bundle.sh > deploy/base/mutatingwebhook-ca-bundle.yaml
 
 delete-certs:
 	@echo 'Deleting mutating controller certs...'
 	kubectl delete secret trace-context-injector-webhook-certs
 
 deploy-config:
 	@echo 'Applying configuration to active cluster...'
-	kubectl apply -f deploy/deployment.yaml
-	kubectl apply -f deploy/mutatingwebhook-ca-bundle.yaml
-	kubectl apply -f deploy/service.yaml
+	kustomize build deploy/overlays/default | kubectl apply -f -
```

Comment:
generally, the default and the base should be the same.  You can provide example patches for extra functionality to show how users could choose to deviate from the default.

Reply:
done